code/controllers/subsystem/metrics.dm

```diff
@@ -0,0 +1,47 @@
+SUBSYSTEM_DEF(metrics)
+	name = "Metrics"
+	wait = 30 SECONDS
+	offline_implications = "Server metrics will no longer be ingested into monitoring systems. No immediate action is needed."
+	runlevels = RUNLEVEL_LOBBY | RUNLEVEL_SETUP | RUNLEVEL_GAME | RUNLEVEL_POSTGAME // ALL THE LEVELS
+	flags = SS_KEEP_TIMING // This needs to ingest every IRL minute
```

Comment:
But it fires once every 30 seconds?

Reply:
Forgot to update this one. We changed the fire rate to every 30 seconds since the tick usage from this was negligible, and it was nice to have more common metrics.